### PR TITLE
Feature/msp menu api

### PR DESCRIPTION
**Pull request by recallsong**
#### What type of this PR

/kind polish

#### What this PR does / why we need it:
move tmc menu apis to msp

#### Specified Reviewers:
@johnlanni @liuhaoyang 

#### Need cherry-pick to release versions?
no

image

### CODE REVIEW

**Review by johnlanni — APPROVED** [review]
wow

File: modules/msp/menu/menu.service.go [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+			for k, v := range params {
+				item.Params[k] = fmt.Sprint(v)
+			}
+		}
+
+		// setup exists
+		isK8s := clusterInfo.IsK8S()
+		for _, child := range item.Children {
+			child.Params = item.Params
+			// 反转exists字段，隐藏引导页，显示功能子菜单
```
Comment (luoyuctl) [inline modules/msp/menu/menu.service.go:105]: internationalization

**Review by luoyuctl — DISMISSED** [review]

File: modules/msp/menu/menu.service.go (hunk 1 above)
Comment (recallsong) [reply modules/msp/menu/menu.service.go:105]: old code comment

**Review by johnlanni — APPROVED** [review]

**Pull request merged**